src/components/table/template.test.js

```diff
@@ -0,0 +1,374 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('table')
+describe('Table', () => {
+
```

Comment:
_Block must not be padded by blank lines_

Reply:
fixed